client/lcd/lcd_test.go

```diff
@@ -730,26 +730,31 @@ func TestProposalsQuery(t *testing.T) {
 	cleanup, _, _, port := InitializeTestLCD(t, 1, []sdk.AccAddress{addrs[0], addrs[1]})
 	defer cleanup()
 
+	depositProcedure := getDepositProcedure(t, port)
+	halfMinDeposit := depositProcedure.MinDeposit.AmountOf("steak").Int64() / 2
+	getVotingProcedure(t, port)
```

Comment:
Is there a value to test the voting and tallying procedures against?

Reply:
Would require a large refactor of that test, which I don't want to do right now 😐

Reply:
Ok np, mind opening an issue then ?